packages/barracuda/0.1.0/docs/README.md

```diff
@@ -0,0 +1,9 @@
+# Barracuda integration
+
+This integration is for Barracuda device's logs. It includes the following
+datasets for receiving logs over syslog or read from a file:
+- `waf` dataset: supports Barracuda Web Application Firewall logs.
+
+### Waf
+
+The `waf` dataset collects Barracuda Web Application Firewall logs.
```

Comment:
So, generally the way this file gets generated in the integrations directory, it contains a listing of all of the fields along with their documentation. Does this need to be consistent here too?